device_calendar/lib/src/device_calendar.dart

```diff
@@ -16,21 +16,26 @@ class DeviceCalendarPlugin {
   DeviceCalendarPlugin._createInstance();
 
   /// Requests permissions to modify the calendars on the device
+  /// TODO: add comment about return value
```

Comment:
Investigate on how to create comments on the code - input parameters and return values